frontend/src/wordInteractions.js

```diff
@@ -48,19 +50,22 @@ function loadWordsFromApi () {
   fetch('http://localhost:3000/word_banks')
     .then(resp => resp.json())
     .then(json => ALL_WORDS = json)
-    .then(chooseRandomWords);
+    .then(() => {
+      for (let i = 0; i < 3; i++) {
+        chooseRandomWord();
+      }
+    });
 }
 
-function chooseRandomWords () {
-  const wordsOnPage = document.getElementsByClassName('untyped');
-  while (wordsOnPage.length < 3) {
-    const location = Math.floor(Math.random() * ALL_WORDS.length);
-    const randomWord = ALL_WORDS[location].word;
-    if (!game.wordsSeen.includes(randomWord)) {
-      addWordToPage(randomWord);
-      game.wordsSeen.push(randomWord);
-    }
+function chooseRandomWord () {
+  let location = Math.floor(Math.random() * ALL_WORDS.length);
+  let randomWord = ALL_WORDS[location].word;
+  while (game.wordsSeen.includes(randomWord)) {
+    location = Math.floor(Math.random() * ALL_WORDS.length);
+    randomWord = ALL_WORDS[location].word;
```

Comment:
The one thing we could do here potentially would just be to remove them from all words and repopulate on a new game, might be faster in the long run. Since this is essentially just a hash the remove could be really fast!

Reply:
mmm good point, I'll work on refactoring that

Reply:
ditto (: